test_runner/performance/test_perf_pgbench.py

```diff
@@ -2,29 +2,113 @@
 from fixtures.zenith_fixtures import PgBin, VanillaPostgres, ZenithEnv
 from fixtures.compare_fixtures import PgCompare, VanillaCompare, ZenithCompare
 
-from fixtures.benchmark_fixture import MetricReport, ZenithBenchmarker
+from fixtures.benchmark_fixture import PgBenchRunResult, MetricReport, ZenithBenchmarker
 from fixtures.log_helper import log
 
+from pathlib import Path
+
+import pytest
+from datetime import datetime
+import calendar
+import os
+import timeit
+
+
+def utc_now_timestamp() -> int:
+    return calendar.timegm(datetime.utcnow().utctimetuple())
+
+
+def init_pgbench(env: PgCompare, cmdline):
+    # calculate timestamps and durations separately
+    # timestamp is intended to be used for linking to grafana and logs
+    # duration is actually a metric and uses float instead of int for timestamp
+    init_start_timestamp = utc_now_timestamp()
+    t0 = timeit.default_timer()
+    with env.record_pageserver_writes('init.pageserver_writes'):
+        env.pg_bin.run_capture(cmdline)
+        env.flush()
```

Comment:
generates:

```
    def flush(self):
>       self.cur.execute("checkpoint")
E       psycopg2.errors.InsufficientPrivilege: must be superuser to do CHECKPOINT
```

Reply:
Ah, of course. Ok, I'll bite the bullet and refactor this too.